lib/widgets/poll.dart

```diff
@@ -69,46 +84,62 @@ class _PollWidgetState extends State<PollWidget> {
           store.users[userId]?.fullName ?? zulipLocalizations.unknownUserName)
         .join(', ');
 
-      return Padding(
-        padding: const EdgeInsets.only(bottom: 5),
```

Comment:
poll: Move vote count paddings inside children of Row

It looks like in this revision, this commit actually moves the padding only inside the first child, and drops the padding from the second one.

    This preserves most of the visuals. Except that when the vote count
    box is shorter than the voter names text, the padding between rows
    is gone.

That seems undesirable. What if we include padding on both children, like in the previous revision?

I think this change in this revision may have been in response to this thread:
https://github.com/zulip/zulip-flutter/pull/939#discussion_r1809681899
but I don't really understand the motivation for removing the padding from the voter names.